### PR TITLE
update testing to work from root, add testing documentation

### PR DESCRIPTION
**Pull request by Ryan-Knowles**

### CODE REVIEW

**Comment by Ryan-Knowles** [comment]
@mruwnik I see my formatter changed a lot of the readme markdown, do we have some .editorconfig for that?

File: tests/align_data/sources/test_stampy.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def test_validate_coda_token():
     with patch("align_data.stampy.stampy.CODA_TOKEN", None):
         with patch("sys.exit") as mock:
             dataset.setup()
-            assert mock.called_once_with(1)
+            mock.assert_called_once_with(1)
```
Comment (mruwnik) [inline tests/align_data/sources/test_stampy.py:12]: [NIT] any specific reason for changing this? I personally prefer the assert keyword, not that it's a problem to change it to this

File: README.md [hunk 2]
```diff
@@ -111,21 +113,15 @@ The log level can be configured with the `LOG_LEVEL` environment variable. The d
...
 
-To update the stampy portion of the dataset, you will need a Coda token. Follow these instructions:
-    1. Go to [coda.io](https://coda.io/)
-    2. Create an account and log in
-    3. Go to the API SETTINGS section of your [account settings](https://coda.io/account), and select `Generate API token`. Give your API token a name, and add the following restrictions:
-       1. Type of restriction: Doc or table
-       2. Type of access: Read only
-       3. Doc or table to grant access to: https://coda.io/d/_dfau7sl2hmG
-    4. Copy this token to your `.env` file: `CODA_TOKEN="<coda_token>"`
+To update the stampy portion of the dataset, you will need a Coda token. Follow these instructions: 1. Go to [coda.io](https://coda.io/) 2. Create an account and log in 3. Go to the API SETTINGS section of your [account settings](https://coda.io/account), and select `Generate API token`. Give your API token a name, and add the following restrictions: 1. Type of restriction: Doc or table 2. Type of access: Read only 3. Doc or table to grant access to: https://coda.io/d/_dfau7sl2hmG 4. Copy this token to your `.env` file: `CODA_TOKEN="<coda_token>"`
```
Comment (mruwnik) [inline README.md:116]: does this still render properly?

**Comment by mruwnik** [comment]
the README is just basic markdown, written manually with no formatter run on it

File: tests/align_data/sources/test_stampy.py (hunk 1 above)
Comment (Ryan-Knowles) [reply tests/align_data/sources/test_stampy.py:12]: It was giving an error, as there is no `mock.called_once_with` method

File: README.md (hunk 2 above)
Comment (Ryan-Knowles) [reply README.md:116]: Yes it appears to, but I don't like my autoformater nuking the existing layout lol. It's poor etiquette.

**Review by mruwnik — APPROVED** [review]

**Pull request merged**